contrib/hack/test-apiserver.sh

```diff
@@ -59,4 +59,9 @@ NO_TTY=1 kubectl delete -f contrib/examples/apiserver/serviceclass.yaml
 NO_TTY=1 kubectl delete -f contrib/examples/apiserver/serviceplan.yaml
 NO_TTY=1 kubectl delete -f contrib/examples/apiserver/instance.yaml
 NO_TTY=1 kubectl delete -f contrib/examples/apiserver/binding.yaml
+
+NO_TTY=1 kubectl create -f contrib/examples/podpresets/db_config.yaml
```

Comment:
Nit: The rest of the commands here are chunked by the verb (create, get, delete), while you chunked by the resource here.

Reply:
I agree, but could roughly argue that these should be separated by APIGroup, though I'd want the filesystem division to match.

Reply:
Is it cool if we get that just-so in a follow-up?  My coding bandwidth is extremely limited atm and @jpeeler is going to be picking up this feature.